### PR TITLE
Updates for 3.15.1

### PR DESCRIPTION
**Pull request by denis-alkhelali**

### CODE REVIEW

File: calico-enterprise_versioned_docs/version-3.15/variables.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const variables = {
   noderunning: 'calico-node',
   rootDirWindows: 'C:\\TigeraCalico',
   registry: 'quay.io/',
-  chart_version_name: 'v3.15.0-1',
+  chart_version_name: 'v3.15.1-0',
```
Comment (denis-alkhelali) [inline calico-enterprise_versioned_docs/version-3.15/variables.js:17]: `0` is a `helmRelease` of releases[0]. It used only in `chart_version_name`, so when i was doing a variables cleanup work i decided to put it directly to `chart_version_name`. Rene have set `helmRelease` to `0` for 3.15.1 
https://github.com/tigera/calico-private/commit/5b7a1ef357130ad75d9adbb8cd4ff73a862ed0f0#diff-488e9b61b18f78da67f2bca98a504b0c83818b8471c60c774f23ed9f7df3f02aR3
@ctauchen

Comment (ctauchen) [reply calico-enterprise_versioned_docs/version-3.15/variables.js:17]: Got it, thanks!

**Comment by ctauchen** [comment]
@rene-dekker FYI

**Review by rene-dekker — APPROVED** [review]
lgtm

**Pull request merged**